src/lib.rs

```diff
@@ -163,6 +163,18 @@ pub mod prelude;
 
 mod core_support;
 
+cfg_if! {
+    if #[cfg(feature = "v1")] {
+        pub mod v1;
+    }
+}
+
+cfg_if! {
+    if #[cfg(feature = "v4")] {
+        pub mod v4;
```

Comment:
The `v4` module probably doesn't need to be public since we don't declare any public items in it, just an `impl` block. Otherwise it'll show up as an empty module in the docs.